rs/rosetta-api/icp_ledger/rosetta-integration-tests/tests/tests.rs

```diff
@@ -112,6 +112,17 @@ impl RosettaTestingClient {
             .unwrap_or_else(|| panic!("Block with id {id:?} not found"))
     }
 
+    async fn block_transaction(
+        &self,
+        block_id: BlockIdentifier,
+        tx_id: TransactionIdentifier,
+    ) -> Result<rosetta_core::response_types::BlockTransactionResponse, Error> {
+        let network = self.network_or_panic().await;
+        self.rosetta_client
+            .block_transaction(network, block_id.clone(), tx_id.clone())
+            .await
+    }
```

Comment:
Is there anything specific to rosetta blocks here? Looking at the code that is called from here, it seems applicable to "regular" blocks also, i.e., blocks containing a single transaction each?

Reply:
Yes, the endpoint block/transaction can be used for RosettaBlocks as well as ICP Blocks. Historically, block/transaction was never really used because there was no point in querying a transaction in a block if there only exists one transaction per block. With RosettaBlocks this has changed however.

Reply:
That makes sense, thanks for the explanation!